api/nodes/views.py

```diff
@@ -40,7 +40,7 @@ def get_default_odm_query(self):
         return (
             Q('is_public', 'eq', True) &
             Q('is_deleted', 'ne', True) &
-            Q('is_dashboard', 'ne', True)
```

Comment:
Hm, I don't think we want to include the Dashboard node by default. That the dashboard node is a `Node` in the database is an implementation detail that should be hidden to users.